server/services/timelineService.test.ts

```diff
@@ -7,6 +7,7 @@ import { EducationAndWorkPlanClient } from '../data'
 import { toTimeline } from '../data/mappers/timelineMapper'
 import aValidTimelineResponse from '../testsupport/timelineResponseTestDataBuilder'
 import aValidTimeline from '../testsupport/timelineTestDataBuilder'
+import config from '../config'
 
 jest.mock('../data/mappers/timelineMapper')
 
```

Comment:
I've changed a lot in this test, which is difficult to make out from the diff (partly because I've changed the grouping and also replaced some of the JSON with builders). It might be easier to view the file.